src/main/java/duke/command/AddCommand.java

```diff
@@ -0,0 +1,39 @@
+package duke.command;
+
+import duke.DukeException;
+import duke.Storage;
+import duke.TaskList;
+import duke.Ui;
+import duke.task.Task;
+
+/**
+ * Represents a command of adding task
+ */
+public class AddCommand extends Command {
+
+    private Task task;
+
+    /**
+     * Constructs an add command
+     *
+     * @param task task to be added
+     */
+    public AddCommand(Task task) {
+        this.task = task;
+    }
+
+    /**
+     * Adds task to task list and saves task list
+     *
+     * @param tasks List of task
+     * @param ui User interface of programme
+     * @param storage Storage of programme
+     * @throws DukeException If error saving file
+     */
```

Comment:
Maybe using the `@inheritDoc` property mentioned in the coding standard across classes extended from `Command.java` would be better in saving lines of code, with added modifications for changed behaviour.